build/common.ps1

```diff
@@ -3,6 +3,10 @@ $DefaultConfiguration = 'debug'
 $DefaultReleaseLabel = 'zlocal'
 $DefaultMSBuildVersion = 15
 
+# The pack version can be inferred from the .nuspec files on disk. This is only necessary as long
+# as the following issue is open: https://github.com/NuGet/Home/issues/3530
+$PackVersion = "3.6.0"
```

Comment:
Add the `common.ps1` file to a list in `UpdateNuGetVersion.ps1` to keep it always updated.